enhancements/CICDv1.md

```diff
@@ -11,36 +11,52 @@ last-updated: 2023-05
 tracking-links: # link to related GitHub issues
   - https://github.com/orgs/sustainable-computing-io/projects/2/views/1?pane=issue&itemId=28750581
 ---
-
+<!-- markdownlint-disable  MD025 -->
```

Comment:
👀 why disable/re-enable the lint here? Is this because of the YAML frontmatter messing with the linter?

Reply:
Yeah it was. 
I will check if I can add more regex to the settings to see if I can get this to work without the disabling of the rule

Reply:
Added some more config to stop the front matter triggering the Single Title Heading rule